README.md

```diff
@@ -2,21 +2,21 @@
 
-- [data_profiling](https://github.com/tuva-health/data_profiling): Runs data quality tests to check for common problems specific to healthcare claims data.
-- [claims_preprocessing](https://github.com/tuva-health/claims_preprocessing): Groups overlapping claims into a single encounter, assigns every claim to 1 of 15 different encounter types and populates core data tables.
-- [cms_chronic_conditions](https://github.com/tuva-health/chronic_conditions): Implements a chronic condition grouper based on ICD-10-CM codes. As a result, it is possible to know whether each patient in your population has any of ~70 different chronic conditions defined for the grouper.
-- [tuva_chronic_conditions](https://github.com/tuva-health/tuva_chronic_conditions): implements a chronic condition grouper created by the Tuva Project which creates ~40 homogeneous and mutually exclusive chronic condition groups on your patient.
-- [pmpm](https://github.com/tuva-health/pmpm): Calculates spend and utilization metrics for your patient population on a per-member-per-month (pmpm) basis.
-- [readmissions](https://github.com/tuva-health/readmissions): Calculates hospital readmission measures.
-- [terminology](https://github.com/tuva-health/terminology): Makes the latest version of many useful healthcare terminology datasets available as tables in your data warehouse. This package is different from the others because it does not build healthcare concepts on top of your data.
+- [data_profiling](https://thetuvaproject.com/data-marts/data-profiling/about): Runs data quality tests to check for common problems specific to healthcare claims data.
+- [claims_preprocessing](https://thetuvaproject.com/data-marts/claims-preprocessing/about): Groups overlapping claims into a single encounter, assigns every claim to 1 of 15 different encounter types and populates core data tables.
+- [chronic_conditions](https://thetuvaproject.com/data-marts/chronic-conditions/about): Two different chronic condition groupers based on ICD-10-CM codes, one using grouping methodology defined by CMS, and another developed by Tuva. 
+- [pmpm](https://thetuvaproject.com/data-marts/pmpm/about): Calculates spend and utilization metrics for your patient population on a per-member-per-month (pmpm) basis.
+- [readmissions](https://thetuvaproject.com/data-marts/readmissions/about): Calculates hospital readmission measures based on CMS methodology.
+- [terminology](https://thetuvaproject.com/terminology/about): Makes the latest version of many useful healthcare terminology datasets available as tables in your data warehouse. This package is different from the others because it does not build healthcare concepts on top of your data.
```

Comment:
We should delete this section or maybe update it to talk about these as data marts.

Reply:
I changed the verbiage to models and left it in with some tweaks, and added core